src/components/Layout/LandingPageLayout/index.tsx

```diff
@@ -5,15 +5,27 @@ import Header from '@components/Header';
 import Footer from '@components/Footer';
 
 function LandingPageLayout() {
-  const contactUsRef = useRef<HTMLDivElement | null>();
+  const contactUsRef = useRef<HTMLDivElement | null>(null);
+  const aboutUsRef = useRef<HTMLDivElement | null>(null);
+  const howWeWorkRef = useRef<HTMLDivElement | null>(null);
 
   const scrollToContactUs = () =>
     contactUsRef.current?.scrollIntoView({ behavior: 'smooth' });
 
+  const scrollToAboutUs = () =>
```

Comment:
Since its practically the same arrow functions you can make only  one and then just forward it to Header component.
Like so: 
const scrollToSection = (sectionRef) => {
  sectionRef.current?.scrollIntoView({ behavior: 'smooth' });
};